docs/deploying.md

```diff
@@ -68,13 +68,11 @@ When deploying automatically, you can’t sign-in in your browser the way you di
 
 To create an API key:
 
-1. Go to [observablehq.com](https://observablehq.com).
-2. In the left sidebar, click **Settings**.
-3. In the right sidebar, click **API / Notebook Keys**.
-4. Click **New API Key**.
-5. Check the **Deploy new versions of projects** checkbox.
-6. Give your key a description, such as “Deploy via GitHub Actions”.
-7. Click **Create API Key**.
+1. Open the [API Key settings](https://observablehq.com/select-workspace?next=api-keys-settings) for your ObservableHQ workspace.
```

Comment:
```suggestion
1. Open the [API Key settings](https://observablehq.com/select-workspace?next=api-keys-settings) for your Observable workspace.
```

We never use “ObservableHQ” to refer to Observable. The `observablehq` name is used solely to avoid conflict in shared, strict global namespaces that we can’t control (e.g., npm scope, DNS, social media handles), not part of our natural language brand identity.